api/config/routes.yaml

```diff
@@ -0,0 +1,3 @@
+#index:
+#    path: /
+#    controller: App\Controller\DefaultController::index
```

Comment:
:+1:

Reply:
This project uses annotations to define routes, then, I will check if I can delete this yalm in the following commits.

Reply:
I don't think a yaml file can have "annotations". I think this is there just for enabling it at a later point.